lib/splitclient-rb/split_config.rb

```diff
@@ -249,6 +251,10 @@ def self.default_redis_url
       'redis://127.0.0.1:6379/0'
     end
 
+    def self.default_redis_namespace
+      "SPLITIO/ruby-#{VERSION}"
```

Comment:
This constant comes from the class in the current namespace (i.e. `SplitIoClient::VERSION`) which is defined here: `lib/splitclient-rb/version.rb`